app/controllers/dispatch.rb

```diff
@@ -0,0 +1,106 @@
+get '/mission' do
+	# this is turning out to be remarkably similar to the index
+
+
+	# view current missions and upcoming
+	# missions (2 partials)
+	# presented w/ btn to mrk mission start
+
+	unresolved_missions = Mission.where(completed: false).sort_by{|mission| mission.step_off}
+
+	@active_missions = unresolved_missions.select do |mission|
+		mission.initiated
+	end
+
+	@upcoming_missions = unresolved_missions.reject do |mission|
+		mission.initiated
+	end
+
+
+# new idea have a list of all uncompleted missions
+	# green: initiated
+	# red:   uninitiated  where ST > Time.now
+	# black: uninitiated where ST < Time.now
+
+	erb :'dispatch/forecast'
+end
+
+
+# get	'/mission/:id' do
+# 	"you landed on GET /mission/#{params[:id]}"
+# 	# view a specific mission
+# 	# presented with the buttons to close dispathes
+# end
+
+
+get '' do
+	# view payloads
+end
+
+
+get '' do
+	# view a specific soldier
+	# include all mission went on
+	# include ph#, wep#, any statuses
+	# presented w/ btn to chng status
+end
+
+
+get '' do
+	# view a specific truck
+	# include all missions
+	# include radio#, any deffencies + statuses
+	# presented with btn to chng defficiences / status
+
+end
+
+
+post '' do
+	# mark a mission as completed...
+	# this could be automatic when last pass/tk
+	# returns though...
+end
+
+
+post '/mission/:id' do
```

Comment:
not sure what this route's purpose is. if we are creating a new mission, then we should `POST` to `/missions`. if we are updating a specific mission and marking the truck/solider as something else, that is probably a `PUT` to `/missions/:id`